README.md

```diff
@@ -1,7 +1,25 @@
-Barefoot Nomad - Making company travel and accomodation easy and convinient.
+Barefoot Nomad - Making company travel and accomodation easy and convenient.
 =======
 
 ## Vision
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
 ---
+
+## Installation and Usage with Docker
```

Comment:
Good work @elikeyz on setting up the docker file.
Things you need to note:
- You can add a link on how to setup docker on the system or short description on how to set it up on the readme.
- Do you necessarily need to add your username, remember this project is a team's project, not a personal project. leaving it this way `docker build -t iceman-backend` is fine

Reply:
I've made the changes